packages/libs/sdk-mixins/src/mixins/descopeUiMixin/constants.ts

```diff
@@ -7,10 +7,10 @@ export const UI_COMPONENTS_URL_KEY = 'base.ui.components.url';
 
 export const UI_COMPONENTS_URL =
   (IS_LOCAL_STORAGE && localStorage.getItem(UI_COMPONENTS_URL_KEY)) ||
-  'https://static.descope.com/npm/@descope/web-components-ui@<version>/dist/umd/index.js';
+  'https://descopecdn.com/npm/@descope/web-components-ui@<version>/dist/umd/index.js';
 
 export const UI_COMPONENTS_FALLBACK_URL =
   (IS_LOCAL_STORAGE && localStorage.getItem(UI_COMPONENTS_URL_KEY)) ||
-  'https://cdn.jsdelivr.net/npm/@descope/web-components-ui@<version>/dist/umd/index.js';
```

Comment:
we still want cdn.jsdelivr.net as fallback

so need to add UI_COMPONENTS_FALLBACK_URL2 for that